articles/errors/libraries/auth0-js/invalid-token.md

```diff
@@ -0,0 +1,29 @@
+---
+public: false
+---
+# Parsing an HS256-Signed ID Token Without an Access Token
+
+**Error Message**: accessToken parameter is not valid
+
+## Why this error occurred
+
+Tokens signed with the HS256 algorithm cannot be properly validated.
+
+Beginning with **auth0.js version 9** and **Lock version 11**, API calls to **/userinfo** must be made with a valid Access Token.
```

Comment:
This is not accurate, /userinfo always required an access token.

The issue here is that the user did not request an access_token. The fix implies requesting the token.

They don't need to change it RS256, but we can suggest them to do that. If they do, and they are using the access_token to call their own APIs, they will need to change the server code to validate the token with RS256 instead of HS256.